### ISSUE TITLE
Getting warning about error showing method candidates 

### ISSUE THREAD

**Issue by malmaud** [JuliaLang/julia#20157]
```julia
julia> x=Nullable(1)
Nullable{Int64}(1)

julia> x+1
ERROR: MethodError: no method matching +(::Nullable{Int64}, ::Int64)WARNING: Error showing method candidates, aborted
```

**Comment by JeffBezanson** [comment]
Fixed by #20119?

**Comment by KristofferC** [comment]
Fixed and tested.
